plugin.video.vtm.go/resources/lib/modules/channels.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+""" Channels module """
+
+from __future__ import absolute_import, division, unicode_literals
+
+from resources.lib.kodiwrapper import TitleItem
+from resources.lib.modules import CHANNELS
+from resources.lib.modules.menu import Menu
+from resources.lib.vtmgo.vtmgo import VtmGo
+
+
+class Channels:
+    """ Menu code related to channels """
+
+    def __init__(self, kodi):
+        """ Initialise object
+        :type kodi: resources.lib.kodiwrapper.KodiWrapper
+        """
+        self._kodi = kodi
+        self._vtm_go = VtmGo(self._kodi)
+        self._menu = Menu(self._kodi)
+
+    def show_channels(self):
+        """ Shows TV channels """
+        product = self._vtm_go.get_product()
+        kids = (product == 'VTM_GO_KIDS')
+
+        # Fetch EPG from API
+        channel_infos = self._vtm_go.get_live_channels()
+
+        listing = []
+        for i, key in enumerate(CHANNELS):  # pylint: disable=unused-variable
+            channel = CHANNELS[key]
+
+            if kids and channel.get('kids') is False:
+                continue
+
+            # Find this channel in the list
+            channel_info = next((c for c in channel_infos if c.key == key), None)
+
+            if channel_info:
+                # Lookup the high resolution logo based on the channel name
+                icon = '{path}/resources/logos/{logo}-white.png'.format(path=self._kodi.get_addon_path(), logo=channel.get('logo'))
+                fanart = '{path}/resources/logos/{logo}.png'.format(path=self._kodi.get_addon_path(), logo=channel.get('logo'))
+
+                context_menu = [(
+                    self._kodi.localize(30052, channel=channel.get('label')),  # Watch live {channel}
+                    'XBMC.PlayMedia(%s)' %
```

Comment:
`PlayMedia` is enough. `XBMC.PlayMedia` is just legacy stuff, you can easily tell how old it is :)